tests/protocol/test_wire.py

```diff
@@ -838,50 +835,18 @@ def http_get_handler(url, *_, **kwargs):
             self.assertTrue(self.is_host_plugin_extension_artifact_request(urls[3]), "The retry request should have been over the host channel")
             self.assertFalse(HostPluginProtocol.is_default_channel, "The default channel should not have changed")
 
-    def test_upload_logs_should_not_refresh_plugin_when_first_attempt_succeeds(self):
```

Comment:
Upload logs does not need the retry logic for 410. I removed that code and its corresponding tests below